rem/aufile/aufile.c

```diff
@@ -278,3 +278,18 @@ size_t aufile_get_length(struct aufile *af, struct aufile_prm *prm)
 
 	return 0;
 }
+
+/**
+ * Set initial playing position of a WAV file in ms
+ *
+ * @param af  Audio-file
+ * @param prm Audio file parameters from aufile_open
+ *
+ * @return position in bytes or (size_t)-1 in case of an error.
+ */
+size_t aufile_set_position(struct aufile *af, struct aufile_prm *prm, size_t pos_ms)
```

Comment:
Normally in `re` all functions except getter return an error code if there are errors possible.

API functions should contain NULL pointer checks.

```c
if (!af || !prm)
    return EINVAL;
```